pkg/vault/contracts/test/MinimalHooksPoolMock.sol

```diff
@@ -2,27 +2,21 @@
 
 pragma solidity ^0.8.24;
 
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { Address } from "@openzeppelin/contracts/utils/Address.sol";
-
 import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePool.sol";
 import { IHooks } from "@balancer-labs/v3-interfaces/contracts/vault/IHooks.sol";
-import { IRouterCommon } from "@balancer-labs/v3-interfaces/contracts/vault/IRouterCommon.sol";
 import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
-import { IVaultMock } from "@balancer-labs/v3-interfaces/contracts/test/IVaultMock.sol";
-import "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
-
-import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/FixedPoint.sol";
-import { ScalingHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/ScalingHelpers.sol";
+import {
+    HookFlags,
+    TokenConfig,
+    LiquidityManagement,
+    AddLiquidityKind,
+    RemoveLiquidityKind
+} from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
```

Comment:
```suggestion
import "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
```

Same here, since this is a mock, and not in the pool-hooks example pkg, maybe it should be a one-liner? Not sure.

Reply:
Yes; I have the IERC20 imported here anyway.